lib/layout.js

```diff
@@ -126,20 +138,43 @@ const PodiumLayout = class PodiumLayout {
     async process(incoming) {
         const s = incoming;
         s.name = this.name;
+        s.view = this._view;
+        s.development = true;
```

Comment:
Same as I commented in the podlet module: https://github.com/podium-lib/podlet/pull/20/commits/bc76d1dc3915b76446f922725590e8ff933a324d#r271044435